index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Linkbrary</title>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="./src/reset.css" />
+    <link rel="stylesheet" href="./src/style.css" />
+  </head>
+  <body>
+    <nav>
+      <a href="index.html">
+        <img src="./images/logo.svg" alt="홈으로 연결된 Linkbrary 로고" />
+      </a>
+      <a class="cta cta-short" href="signin.html">
+        <span>로그인</span>
+      </a>
+    </nav>
+
+    <header>
+      
+      <div class="hero-header">
+        <h1 class="slogan">
+          <span class="slogan-gradient background-clip-text"> 세상의 모든 정보</span>를
```

Comment:
의도한 whitespace가 아니라면 제거되면 좋을 것 같아요 :)

```suggestion
          <span class="slogan-gradient background-clip-text">세상의 모든 정보</span>를
```